src/dojos/cd-dojo-details.vue

```diff
@@ -106,7 +106,17 @@
     },
     computed: {
       address() {
-        return !this.dojoDetails.address1 ? undefined : [
+        if (!this.dojoDetails.address1) {
+          return undefined;
+        } else if (this.dojoDetails.address1.includes(this.dojoDetails.countryName)) {
+          return this.dojoDetails.address1;
+        } else if (this.dojoDetails.address1.includes(this.dojoDetails.placeName)) {
+          return [
+            this.dojoDetails.address1,
+            this.dojoDetails.countryName,
+          ].join(', ');
+        }
+        return [
```

Comment:
Okay, I did think the function was too long. I was just trying to account for when address1 doesn't include the place or country because some dojos seem to be that way. I will just use address1 instead :)